packages/capabilities/bundle/config/upstream/tanzu-capabilities-controller-manager.yaml

```diff
@@ -9,6 +9,8 @@ metadata:
     app: tanzu-capabilities-manager
   name: tanzu-capabilities-controller-manager
   namespace: #@ data.values.namespace
+  annotations:
+    kapp.k14s.io/disable-default-label-scoping-rules: ""
```

Comment:
There are a couple of ways for migrating resources from CRS to kapp. Link to the blog: https://carvel.dev/blog/migrate-existing-resources-to-a-new-kapp-app/ . I went with the option 1 as option 2's strategy is to delete and create the resource again. The only consequence of option 1 is other app cannot have a deployment with label selector as `app: tanzu-capabilities-manager`, it is very unlikely for some other app to have same label selector